data/artwork.json

```diff
@@ -309,5 +309,19 @@
       "github": "Arc-blroth"
     },
     "license": "CC-BY-SA-4.0"
+  },
+  {
+    "date": "2022-08-03",
+    "image": "/images/artwork/snow_den.png",
```

Comment:
This seems pointing to a wrong file?

Looks like you forgot to commit the actual image file?

Reply:
My bad

Reply:
Done, add image in `./images/artwork/`